dali/test/python/test_operator_multipaste.py

```diff
@@ -130,15 +141,13 @@ def prepare_cuts(
         below_zero = np.random.randint(2) == 0
         change_dim_idx = np.random.randint(dim)
         if below_zero:
-            (in_anchors_l if change_in else out_anchors_l)[clip_out_idx][clip_in_idx][change_dim_idx] = \
-                np.int32(np.random.randint(5) - 5)
+            (in_anchors_l if change_in else out_anchors_l
+             )[clip_out_idx][clip_in_idx][change_dim_idx] = np.int32(np.random.randint(5) - 5)
```

Comment:
Obfuscated Python contest?
```suggestion
            anchors = in_anchors_l if change_in else out_anchors_l
            anchors[clip_out_idx][clip_in_idx][change_dim_idx] = np.random.randint(-5, 0, dtype=np.int32)
```